guide/the-user-interface.md

```diff
@@ -225,34 +228,37 @@ because the far plane is set to infinity. The recorded image is displayed on the
 camera's near plane. More information about this concept is available in the
 OpenGL documentation.
 
-The `Show Distance Sensor rays` allows you to display, or to hide, the rays
+The `Show DistanceSensor Rays` allows you to display, or to hide, the rays
 casted by the distance sensor devices. These rays are drawn as red lines (which
 become green beyond collision points). Their length corresponds to the maximum
 range of the device.
 If the distance sensor device is disabled or the first measurement is not available yet, the rays will be drawn in gray.
 
-The `Show Light Sensor rays` allows you to display, or to hide, the rays casted
+The `Show LightSensor Rays` allows you to display, or to hide, the rays casted
 by the light sensor devices. These rays are drawn as yellow lines.
 If the light sensor device is disabled or the first measurement is not available yet, the rays will be drawn in gray.
 
-The `Show Lights` allows you to display, or to hide, the lights (including
-PointLights and SpotLights). DirectionalLights aren't represented. PointLights
-and SpotLights are represented by a colored circle surrounded by a flare.
+The `Show Light Positions` allows you to display, or to hide, the position of lights (including PointLights and SpotLights).
```

Comment:
It would simplify the sentence to extract the parenthesis content:

```
The `Show Light Positions` allows you to display, or to hide, the position of the PointLights and SpotLights nodes.
```